### DISCUSSION TITLE
AD410 Doorbell "In Call"

### DISCUSSION THREAD

**Discussion by esmoyer** [blakeblackshear/frigate#5342] [General]
So, this took me a good few days and a ton of trial and error, packet captures, firewall configs, etc. but I found that adding the AD410 to Frigate causes it to think it's in an active "call". (ie the ring light turns dark blue and the app says "someone else is on the call"). I noticed this not too long ago, so I wonder if it might have to do with the new restream/go2rtc.

When I comment it out of Frigate and restart, the doorbell goes back to normal (light blue/green ring light). Normally this wouldn't be an issue, but when the doorbell is in a call, then the button doesn't work, kind of defeating the purpose of the doorbell. Anyone else with an AD410 experience this? 

I am going to experiment and try a different RTSP URL as maybe the one I am using is what is used for "calls".

**Comment by NickM-27** [comment]
Yes, this is the case. Simply add `#backchannel=0` to stop it. It's because go2rtc supports two way talk with the ad410 and (currently) it's enabled by default.

**Reply by esmoyer** [reply]
Thanks, Nick. I just found: https://github.com/blakeblackshear/frigate/issues/5020 as well and was going to comment. Can't believe how much time I spent trying to figure that out!
